.buildkite/pipeline.yml

```diff
@@ -68,10 +68,38 @@ steps:
     command: "goss validate --format documentation"
     timeout_in_minutes: 5
     agents:
-      stack: "buildkite-aws-stack-test-linux-${BUILDKITE_BUILD_NUMBER}"
-      queue: "testqueue-linux-${BUILDKITE_BUILD_NUMBER}"
+      stack: "buildkite-aws-stack-test-linux-amd64-${BUILDKITE_BUILD_NUMBER}"
+      queue: "testqueue-linux-amd64-${BUILDKITE_BUILD_NUMBER}"
     depends_on: "linux-launch"
 
+  - id: "packer-linux-arm64"
+    name: ":packer: :linux: :arm:"
+    command: .buildkite/steps/packer.sh linux arm64
+    timeout_in_minutes: 60
+    retry: { automatic: { limit: 3 } }
+    agents:
+      queue: "${BUILDKITE_AGENT_META_DATA_QUEUE}"
+    depends_on:
+      - "lint"
+      - "bats-tests"
+
+  - id: "linux-arm64-launch"
+    name: ":cloudformation: :linux: :arm: Launch"
```

Comment:
oh, turns out we don't support the `:arm:` emoji.

Reply:
hm! there is one that might work for an emoji. i can make a pull to buildkite/emojis

![image](https://user-images.githubusercontent.com/30171259/97524016-c654e100-19f7-11eb-8ef5-f02b20d7818f.png)

Reply:
Maybe we're  restricted in using it? I'm not sure what our policies on corporate emjois are

https://www.arm.com/company/policies/trademarks/guidelines-corporate-logo

> If you do not have permission to use the Arm logo please refer to the Arm Logo Permission Requests page.
>
> You do not need to seek permission if:
>
>    You already have written permission from Arm.
>    You are a journalist, media outlet or blog using the logo for editorial purposes.
>    You are a charity or other not-for-profit organisation that is being supported by Arm.

Reply:
I imagine it's fair use of some sort, and it'll be fine. We probably haven't added it because nobody has suggested/done it. _I am not a lawyer etc etc._

Hmm.

> Do not use the logo or logotype as an element in titles, headlines or text.
> Do not place competing visual elements (including but not limited to other trademarks) close to the logo.

It'd be hard to stick to those. 🤷‍♂️

I've submitted a request to Arm via their Arm Logo Permission Requests page…

> Buildkite is a Continuous Integration platform. Our customers increasingly build software for Arm-based processors. We would like permission to add the Arm corporate logo to our emoji repository[1]. This emoji would allow software engineers using our platform to succinctly indicate that a particular software build target is for Arm processors.
> [1]: https://github.com/buildkite/emojis